testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -5885,7 +5885,9 @@ private static void checkIf(SqlOperatorFixture f) {
 
     final SqlOperatorFixture f = f0.withLibrary(SqlLibrary.SPARK);
     f.checkScalar("array_compact(array[null, 1, null, 2])", "[1, 2]",
-        "INTEGER ARRAY NOT NULL");
+        "INTEGER NOT NULL ARRAY NOT NULL");
+    f.checkScalar("array_compact(array[1, 2])", "[1, 2]",
+        "INTEGER NOT NULL ARRAY NOT NULL");
     f.checkScalar("array_compact(array[null])", "[]",
         "NULL ARRAY NOT NULL");
     f.checkScalar("array_compact(array(null))", "[]",
```

Comment:
Two same tests for `array_compact(array[null])`?

Reply:
@NobiGo thank for reviewing. Yes, it was introduced by the previous commit/PR, and I can remove it in this PR? WDYT?